build/ublue-os-just/40-nvidia.just

```diff
@@ -59,15 +59,46 @@ configure-nvidia ACTION="prompt":
     fi
 
 # Switch between Nvidia image and NVK
-toggle-nvk:
+toggle-nvidia ACTION="prompt":
     #!/usr/bin/bash
     CURRENT_IMAGE=$(rpm-ostree status -b --json | jq -r '.deployments[0]."container-image-reference"' | sed -E 's/^.+\/(.+:.+)$/\1/')
     CURRENT_URI=$(rpm-ostree status -b --json | jq -r '.deployments[0]."container-image-reference"' | sed -E 's/^(.+\/).+:.+$/\1/')
     NEW_IMAGE=$CURRENT_IMAGE
+
     if [[ "$CURRENT_IMAGE" =~ "nvidia" ]]; then
-      NEW_IMAGE=${CURRENT_IMAGE/-nvidia/}
+      IS_CURRENTLY_NVIDIA=true
+      echo "Current nvidia driver: Nvidia (From Nvidia Inc, Proprietary)"
     else
-      NEW_IMAGE=${CURRENT_IMAGE/:/-nvidia:}
+      IS_CURRENTLY_NVIDIA=FALSE
+      echo "Current nvidia driver: NVK (Open Source, MESA)"
+    fi
+
```

Comment:
and this empty line should be removed as they break just highlighting in editors like vscode from my experience (which sucks)